src/app/@ansyn/ansyn/modules/overlays/components/overlay-overview/overlay-overview.component.ts

```diff
@@ -107,10 +112,9 @@ export class OverlayOverviewComponent implements OnInit, OnDestroy {
 			if (!hoveredElement) {
 				return;
 			}
-			const hoveredElementBounds: ClientRect = hoveredElement.getBoundingClientRect();
-			this.myCurrentWidth = (this.el.nativeElement as HTMLElement).offsetWidth;
-			this.left = customElement ? this.isRTL ? hoveredElementBounds.left - this.myCurrentWidth : hoveredElementBounds.right : this.getLeftPosition(hoveredElementBounds.left);
-			this.top = hoveredElementBounds.top + (customElement ? hoveredElementBounds.height : 0);
+			const { left, height, top }: ClientRect = hoveredElement.getBoundingClientRect();
+			this.left = left - (customElement ? 275 : 150);
```

Comment:
Why this change?
Please don't use explicit numbers, but calculations, as in master branch.